src/components/ButtonMenu.jsx

```diff
@@ -48,9 +49,11 @@ export default function ButtonMenu({
           <MenuItem
             key={action.id}
             onClick={e => {
-              action.onClick(e);
+              if (action.onClick) action.onClick(e);
               handleClose();
             }}
+            component={action.href ? RouterLink : undefined}
```

Comment:
I don't understand why this is necessary.

Reply:
ahh, so that we are rendering a link instead of a button in the menu!